libs/shared/package.json

```diff
@@ -38,7 +38,6 @@
     }
   },
   "dependencies": {
-    "@clerk/clerk-sdk-node": "^5.0.19",
```

Comment:
This should have been removed earlier, it's no longer depended on by `@novu/shared`.